python/rascaline/tests/utils/correlate_density.py

```diff
@@ -151,6 +150,25 @@ def get_norm(tensor: TensorMap):
     return norm
 
 
+def get_max_angular(density: TensorMap, calculator: DensityCorrelations):
+    key_metadata = _precompute_keys(
+        density.keys,
+        density.keys,
+        n_iterations=calculator._n_iterations,
+        selected_keys=calculator._selected_keys,
+        skip_redundant=calculator._skip_redundant,
+    )
+    return max(
+        _dispatch.max(density.keys.column("spherical_harmonics_l")),
+        max(
+            [
+                int(_dispatch.max(mdata[2].column("spherical_harmonics_l")))
+                for mdata in key_metadata
+            ]
+        ),
+    )
+
+
```

Comment:
I think this a remain of prototyping, I think don't use this anywhere, but double check
```suggestion
```